src/v1/controllers/news.controller.js

```diff
@@ -0,0 +1,54 @@
+const catchAsync = require("../../utils/catchAsync");
+const {PreferenceService} = require("../services");
+require('dotenv').config()
+const axios = require('axios')
+
+let everythingEndpoint = 'https://newsapi.org/v2/everything'
+let topHeadlinesEndpoint = 'https://newsapi.org/v2/top-headlines'
+
+const getNews = catchAsync(async (req, res) => {
+    let {user} = req
+    let preferences = await PreferenceService.getUserPreferences(user.id)
+    console.log(preferences)
+    let userKeywords = preferences || []
+    console.log("userKeywords")
+    if (userKeywords.length === 0) {
+        userKeywords = ['bitcoin']
```

Comment:
Is this the default preference to be used in the feed? If yes, it can be fetched from a different service I think. That service layer can read it from the config for now. A similar approach will make it easy to have the default preference be, let's say, machine learning driven for example in the future.